README.rst

```diff
@@ -0,0 +1,66 @@
+Django Zendesk Tickets
+======================
+
+Installation
+------------
+
+At the moment, the only way to install the library is from github
+
+.. code-block:: bash
+
+    pip install git+https://github.com/ministryofjustice/django-zendesk-tickets.git
+
+Settings
+--------
+
+.. code-block:: python
+
+    ZENDESK_BASE_URL=http://zendesk.example.com
+    ZENDESK_API_USERNAME=...
+    ZENDESK_API_TOKEN=...
+    ZENDESK_REQUESTER_ID=...
+    ZENDESK_GROUP_ID=...
```

Comment:
We don't need it for our use, but I wonder if it might make more sense for these to be settable in the `urls.py` blocks or similar? That way, a service could work with different ZenDesk accounts/users/groups/whatever easily in one app.